_pkgdown.yml

```diff
@@ -1,5 +1,14 @@
 url: https://r.duckdb.org/
 
+template:
+  bootstrap: 5
+
+authors:
+  Hannes Mühleisen:
+    href: "http://hannes.muehleisen.org/"
+  Mark Raasveldt:
+    href: "https://mytherin.github.io/"
+
```

Comment:
Add https://krlmlr.info/ for myself too?

Reply:
You are not listed as an author (yet?)!